app/Http/Controllers/FollowsController.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace App\Http\Controllers;
+
+
+use App\Models\User;
+use App\Models\Follow;
+use Illuminate\Support\Facades\Auth;
+
+class FollowsController extends Controller
+{
+    //フォロー
+    public function follow(User $user) 
+    {
+        $login_user = auth()->user();
+        $is_following = $login_user->is_following($user->id);
+        // フォローしているか
+        if(!$is_following) {
+            // フォローしていなければフォローする
+            $login_user->follow($user);
+        }
+        
+        return back();
+    }
+
+    //フォロー解除
+    public function unfollow($id)  
+    {
+        $user = User::find($id);
+        $login_user = auth()->user();
+        $is_following = $login_user->is_following($user->id);
+        if($is_following)
+        {
+            $login_user->unfollow($user);
+        }
+       
+        return back();
+    }
+
+    public function followings($id)
+    {  
+      $user = User::find($id);
+      $login_user = auth()->user();
+      $followings = $user->follows()->get();
+      return view('users.following', [
+            'followings'  => $followings ,
+            'user' => $user ,
+            'login_user' => $login_user,
+       ]);
+    }
+
+    public function followers($id)
+    {
+        $user = User::find($id);
+        $login_user = auth()->user();
+        $followers = $user->followers()->get();
+        return view('users.follower', [
+            'followers'  => $followers,
+            'login_user' => $login_user,
+            'user' => $user ,
```

Comment:
`,` の前に半角スペースはなしでOKです
```suggestion
            'user' => $user,
```
(他の箇所も合わせて対応をお願いします)